libp2p/src/main/kotlin/io/libp2p/pubsub/gossip/GossipRouter.kt

```diff
@@ -349,7 +349,7 @@ open class GossipRouter(
     }
 
     private fun processPrunePeers(peersList: List<Rpc.PeerInfo>) {
-        peersList.shuffled(random).take(params.maxPrunePeers)
+        peersList.shuffled(random).take(params.maxPeersAcceptedInPruneMsg)
```

Comment:
nit: that `take()` call seems to have no effect (as a message exceeding the limit should be dropped before), but I would probably leave it just to be safe in case of future modifications